### ISSUE TITLE
Enum with underlying type long

### ISSUE THREAD

**Issue by alec-anikin** [neisbut/Npgsql.Bulk#15]
NpgsqlBulkCodeBuilder line 158 
```
else if (mi.ReturnType.GetTypeInfo().IsEnum)
{
     ilOut.Emit(OpCodes.Ldc_I4_S, (int)info.NpgsqlType);
     ilOut.Emit(OpCodes.Call, writeMethodFull.MakeGenericMethod(typeof(Int32)));
}
```
There should be check for enum:long. 
Thanks for useful library!

**Comment by msvprogs** [comment]
Not only long but all possible integer types should be supported. Just pass Enum.GetUnderlyingType() instead of hardcoded Int32.

**Comment by neisbut** [comment]
Hi @alec-anikin, @msvprogs, thanks for usefull suggestion. I've implemented this in 0.6.9 release. Please check.
